go/libkb/kbsig.go

```diff
@@ -150,7 +150,13 @@ func (u *User) ToTrackingStatementSeqTail() *jsonw.Wrapper {
 func (u *User) ToTrackingStatement(w *jsonw.Wrapper, outcome *IdentifyOutcome) (err error) {
 
 	track := jsonw.NewDictionary()
-	track.SetKey("key", u.ToTrackingStatementKey(&err))
+	var keyErr error
+	key := u.ToTrackingStatementKey(&keyErr)
```

Comment:
Can this be cleaned up a bit?  I think it's better to not call this if `!u.HasActiveKey()`?